src/style.scss

```diff
@@ -27,9 +27,24 @@
 //   ...
 // }
 
-.testClass {
-  color:  var(--auro-color-border-error-on-light);
-  border: 1px solid var(--auro-color-border-error-on-light);
-  display: inline-block;
-  padding: var(--auro-text-body-size-default);
+.badge {
+  align-items: center;
+  background-color: var(--auro-color-ui-default-on-light);
+  border-radius: 22px;
+
+  display: inline-flex;
+  justify-content: center;
+  padding-top: 10px;
```

Comment:
These paddings should use the auro-spacing values. The typical auro spacing is multiples of 24. E.g. an icon is 24px and the auro-button is 48.